beacon-chain/sync/subscriber_light_client.go

```diff
@@ -0,0 +1,44 @@
+package sync
+
+import (
+	"context"
+
+	"github.com/OffchainLabs/prysm/v6/beacon-chain/core/feed"
+	statefeed "github.com/OffchainLabs/prysm/v6/beacon-chain/core/feed/state"
+	lightclientTypes "github.com/OffchainLabs/prysm/v6/consensus-types/light-client"
+	"google.golang.org/protobuf/proto"
+)
+
+func (s *Service) lightClientOptimisticUpdateSubscriber(_ context.Context, msg proto.Message) error {
+	update, err := lightclientTypes.NewWrappedOptimisticUpdate(msg)
+	if err != nil {
+		return err
+	}
+
+	log.Debug("saving newly received light client optimistic update")
```

Comment:
```suggestion
	log.Debug("Saving newly received light client optimistic update")
```

Also log some more specific data on the updates (slot, root,etc)